.github/workflows/build.yml

```diff
@@ -44,14 +45,21 @@ jobs:
                   Stats.html
                   Conformance.Darwin.html
                   Performance.Darwin.html
+                  nano Performance.csv
```

Comment:
What does this do?

Reply:
This was just for when I was working on performance in Jsonifier.